appinfo/Migrations/Version20191109111104.php

```diff
@@ -0,0 +1,39 @@
+<?php
```

Comment:
`@author Semih Serhat Karakaya <karakayasemi@itu.edu.tr>`, etc header is missing

Reply:
Header has been added.